.pre-commit-config.yaml

```diff
@@ -4,23 +4,32 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.4.0 # Get the latest version from: https://github.com/pre-commit/pre-commit-hooks/releases
     hooks:
-      - id: end-of-file-fixer
-      - id: trailing-whitespace
       - id: check-yaml
       - id: detect-aws-credentials
         args: ["--allow-missing-credentials"]
   - repo: https://github.com/antonbabenko/pre-commit-terraform
     rev: v1.77.1 # Get the latest version from: https://github.com/antonbabenko/pre-commit-terraform/releases
     hooks:
       - id: terraform_fmt
-      - id: terraform_docs
       - id: terraform_validate
       - id: terraform_tfsec
       - id: terraform_checkov
+  - repo: local
+    hooks:
+      - id: docs
+        name: docs
+        entry: make
+        args: [ 'docs' ]
+        language: system
   - repo: https://github.com/gitguardian/ggshield
     rev: v1.14.4
     hooks:
       - id: ggshield
         language: python
         stages: [commit]
         args: [ 'secret', 'scan', 'pre-commit' ]
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0 # Get the latest version from: https://github.com/pre-commit/pre-commit-hooks/releases
```

Comment:
Is this our new standard?

Reply:
should be like this

https://github.com/honestbank/.github/blob/main/others/pre-commit-config-terraform.yaml

will update, but yes end of line and white space fixes has been moved to the  end